dumper-integration-tests/redshift-tests/src/main/java/com/google/jdbc/JdbcUtil.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.jdbc;
+
+import static com.google.base.TestConstants.TRAILING_SPACES_REGEX;
+
+import java.math.BigDecimal;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.ZonedDateTime;
+import java.util.Calendar;
+import java.util.TimeZone;
+
+/**
+ * A helper class for checking Null values returned by executing SELECT request against a database.
+ */
+public final class JdbcUtil {
+
+  private JdbcUtil() {}
+
+  /**
+   * @param rs A row with SELECT results.
+   * @param column Database column name.
+   * @return String or an empty string if null.
+   */
+  public static String getStringNotNull(ResultSet rs, String column) throws SQLException {
+    String string = rs.getString(column);
+    return rs.wasNull() ? "" : TRAILING_SPACES_REGEX.matcher(string).replaceFirst("");
+  }
+
+  /**
+   * @param rs A row with SELECT results.
+   * @param column Database column name.
+   * @return int or 0 if null.
+   */
+  public static int getIntNotNull(ResultSet rs, String column) throws SQLException {
+    int intValue = rs.getInt(column);
+    return rs.wasNull() ? 0 : intValue;
+  }
+
+  /**
+   * @param rs A row with SELECT results.
+   * @param column Database column name.
+   * @return long or 0L if null.
+   */
+  public static long getLongNotNull(ResultSet rs, String column) throws SQLException {
+    long longValue = rs.getLong(column);
```

Comment:
This is primitive so it can't be null.

Reply:
`ResultSet.getLong` already does the `null` to `0` coalesce, so the next line is redundant. This holds for all `ResultSet.get*` which return a primitive type.

Reply:
Done.